### DISCUSSION TITLE
Would like to access design of pedalino 1 as a sponsor

### DISCUSSION THREAD

**Discussion by manlikewhisper** [alf45tar/PedalinoMini#517] [General]
.

**Comment by alf45tar** [comment]
You subscribed for PedalinoMini 4 STL files (9 USD).
PedalinoMini Case 1 is available for all sponsors starting from 1 USD/month including firmware update.
Do you prefer to move your contribution to 9 months subscription which not included PedalinoMini 4/6/8?

**Reply by jaylfc** [reply]
Just wondering how do I access the private repo, I used to have access but my sponsorship stopped due to new bank card. Updated now, thanks for your awesome work :)

**Reply by alf45tar** [reply]
Added you again to private repo.

**Reply by jaylfc** [reply]
Thanks!

**Comment by manlikewhisper** [comment]
Wow. No, I'll just edit the file. Thanks.

**Comment by manlikewhisper** [comment]
Congratulations by the way!

**Comment by markus625** [comment]
I've just become a sponser, am I supposed to get access to a private repository or something, not sure if my access has changed from previously

**Comment by markus625** [comment]
Just got your invitation by the way thanks 👍

**Comment by MiqViq** [comment]
Hi @alf45tar, do I get a full access to all private assets in Pedalino Mini repo with one-time-payment of $25.
Not sure as the description just tells: "36 months of new firmware private repository access".

**Comment by alf45tar** [comment]
As of version 4.0.0, PedalinoMini™ has transitioned to a closed-source model. Active sponsors will receive 2 license credits for each euro of subscription. They can be used for any device and firmware version.

With a payment of $25 you will obtain 50 license credits to be spend in the next 36 months. You can activate 50 devices with one versions or 1 device with the next 50 versions or any combination of them. For example you can activate 5 devices with 10 different firmware version.

You will not obtain the source-code of the new firmware versions or STL files of PedalinoMini™ 4/6/8.

**Reply by zavorra** [reply]
@alf45tar  where was this policy transition announced?

**Reply by alf45tar** [reply]
https://github.com/alf45tar/PedalinoMini/blob/master/doc/Licensing.md#-licensing--activation

Last question of FAQ

**Reply by zavorra** [reply]
So, how do I as a sponsor know what is my current "license credit"?

EDIT: and do you have a Path Forward to share with us in terms of features you are going to implement?

**Reply by zavorra** [reply]
Also, you mention on the FAQ:

_1. Lack of Meaningful Community Contributions
While PedalinoMini™ was open source, the vast majority of development — including features, fixes, and support — came from a single maintainer. Community contributions were limited, and maintaining open repositories offered little return in terms of collaboration._

TBH I tried sharing suggerstion, request and patches, but you never bothered answering...

**Reply by alf45tar** [reply]
> So, how do I as a sponsor know what is my current "license credit"?
> 
> EDIT: and do you have a Path Forward to share with us in terms of features you are going to implement?

Thanks for you long term support as a sponsor. As of today you can redeem up to 42 license credits until you sponsorship remain active. 
I don't have a roadmap. I am an hobbyist and this is not my primary job.

**Reply by alf45tar** [reply]
To activate you devices send me the MAC addresses.

**Reply by alf45tar** [reply]
> Also, you mention on the FAQ:
> 
> _1. Lack of Meaningful Community Contributions While PedalinoMini™ was open source, the vast majority of development — including features, fixes, and support — came from a single maintainer. Community contributions were limited, and maintaining open repositories offered little return in terms of collaboration._
> 
> TBH I tried sharing suggerstion, request and patches, but you never bothered answering...

We are on the same page. If not me none will answering you. Community contributions were limited, and maintaining open repositories offered little return in terms of collaboration.
